spec.bs

```diff
@@ -8,38 +8,40 @@ Repository: dylanvanassche/rdf-ts-snippets-spec
 URL: https://dylanvanassche.github.io/rdf-ts-snippets-spec/
 Editor: Gerald Haesendonck
 Editor: Dylan Van Assche
-Abstract: A RDF Time Series Snippet is a subset of a Time Series described in RDF. [=Snippet=]s aim to be more compact compared to RDFS Collection and Containers for describing Time Series by levering the Common Data Types.
+Editor: [Pieter Colpaert](https://pietercolpaert.be/#me)
+Abstract: This document specifies how time series data with with only a few changing values
+  can be described in RDF.
 Markup Shorthands: markdown yes, css no
 </pre>
 
-# Introduction # {#intro}
+# Abstract # {#abstract}
```

Comment:
This breaks BikeShed, if Abstract key is used in the metadata above, this section is auto-created and cannot be mentioned twice.

Reply:
I don’t remember adding this myself -- I’ll amend it!